client/src/components/gammaSelection.tsx

```diff
@@ -0,0 +1,243 @@
+  const getData = () => {
+    let pdfs = [];
+    let gamma_data;
+
+    setLoading(true);
+    fetch(`/data/${gamma}`, {
+      method: "POST",
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        gamma_data = res;
+        // iterating over each solution to fetch its plot
+        for (let i = 0; i < gamma_data[`${gamma}`].length; i++) {
+          fetch("/pdf", {
+            method: "POST",
+            body: JSON.stringify(gamma_data[`${gamma}`][i]["path"]),
+          })
+            .then((res) => res.blob())
+            .then((blob) => {
+              let objectURL = URL.createObjectURL(blob);
+              pdfs.push(objectURL);
+              // rewriting OS path to PDF to the blob URL
+              gamma_data[`${gamma}`][i]["path"] = objectURL;
+              // this line is needed for images to render on UI
+              setPDF(objectURL);
+            });
+        }
+        setImages(pdfs);
+        setData(gamma_data);
+      })
+      .catch((err) => {
+        setError(err);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  };
+
+  // submits selected parameters to database
+  const handleFormSubmit = () => {
+    // casting form fields, which are Strings, to Number type before submitting
+    let gamma_submit = Number(gamma);
+    let cellularity_submit = Number(cellularity);
+    let ploidy_submit = Number(ploidy);
+    // submit to DB
+    console.log({
+      "selected-folder": selectedFolder,
+      gamma: gamma_submit,
+      cellularity: cellularity_submit,
+      ploidy: ploidy_submit,
+    });
+    // after submission to db, show this alert
+    toast.success("Submitted to database", {
+      position: "top-right",
+      autoClose: 3000,
+      hideProgressBar: true,
+      closeOnClick: true,
+      pauseOnHover: false,
+    });
+  };
+
+  return (
+    <>
+      <Container
+        style={{ paddingRight: "3vw", paddingLeft: "3vw", paddingTop: "2vh" }}
+      >
+        <p>
+          Select a folder to get started.{" "}
+          {selectedFolder && (
+            <>
+              Use the slider to see primary solutions for different gammas.
+              Choose a value from the Gamma dropdown to see the all solutions
+              for that gamma.
+            </>
+          )}
+        </p>
+      </Container>
+      <Container style={{ paddingRight: "3vw", paddingLeft: "3vw" }}>
+        <Row>
+          <Col>
+            <DropdownMenu
+              label="Select Folder"
+              value={selectedFolder}
+              setValue={setSelectedFolder}
+              data={folders}
+            />
+          </Col>
+
+          {selectedFolder && (
+            <Col>
+              <DropdownMenu
+                label="Gamma"
+                value={gamma}
+                setValue={setGamma}
+                data={gammaOptions}
+              />
+            </Col>
+          )}
+        </Row>
+
+        {/* <Row>
```

Comment:
Try to avoid committing large blocks of commented-out code -- we can always get old code back from Git history if necessary.

Reply:
This code is for the two form fields and submit button. We decided to comment this out so that when CouchDB is set up, we can just uncomment and it will be good to go. Would you prefer I delete it instead?

Reply:
In that case you can leave it there. But put another comment to explain what the commented-out block is for. As I've just demonstrated, people forget otherwise. :-)